src/libmain.c

```diff
@@ -1287,8 +1289,6 @@ static gboolean do_main_quit(void)
 
 	geany_debug("Quitting...");
 
-	main_status.quitting = TRUE;
```

Comment:
Why?

Reply:
Because it's already set in https://github.com/geany/geany/blob/4264d0c507abc32c4e9f13aebaaec53d0c9e376b/src/libmain.c#L1405

and the extra assignment here is confusing and makes the impression that

```
configuration_save();
project_close();
document_close_all();
```

above it are made without this flag set.